### PR TITLE
Upgrade .NET Interactive

### PR DESCRIPTION
**Pull request by colombod**
This PR upgrades .NET Interactive dependencies

### CODE REVIEW

**Comment by luisquintanilla** [comment]
@LittleLittleCloud @JakeRadMSFT can you please take a look at this PR to help unblock @colombod 

@michaelgsharp can we get an approval please. 

Thanks

**Review by JakeRadMSFT — APPROVED** [review]

**Comment by colombod** [comment]
when this is going to be merged and release on nuget?

**Pull request merged**
